### PR TITLE
FEAT:- Added Homepage to my quiz app

### PR DESCRIPTION
**Pull request by zoef-yt**
In this pr, I created a homepage of my quiz app

# Files to Check 
All

[Preview Link](https://deploy-preview-1--quiz-e-games.netlify.app/)

### CODE REVIEW

File: logic/theme-changer.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+if (currentTheme === 'light') {
+	document.documentElement.setAttribute('data-theme', 'light');
+	document.querySelectorAll('.theme-changer-icon').forEach(function (item) {
+		item.src = '/assets/svg/moon.svg';
+	});
+} else {
+	document.documentElement.setAttribute('data-theme', 'dark');
+	document.querySelectorAll('.theme-changer-icon').forEach(function (item) {
+		item.src = '/assets/svg/sun.svg';
+	});
```
Comment (abhishektanwar) [inline logic/theme-changer.js:13]: why are we not using getElementsByClassName[0] here when we know that there is only one icon to toggle theme . Using this eliminates forEach.

**Review by abhishektanwar — APPROVED** [review]
Code is clean and understandable . Good UI ,great work.

**Review by srishtigautam21 — APPROVED** [review]
The UI looks awesome
The code looks good to me
Good to merge

**Review by sudipkundu999 — APPROVED** [review]
The UI looks soo good.
No suggestions from my side for the code.
Good work

File: logic/theme-changer.js (hunk 1 above)
Comment (zoef-yt) [reply logic/theme-changer.js:13]: Can do this as there will be many "theme-changer-icon" on multiple pages this crash.

**Pull request merged**